system_files/shared/usr/share/ublue-os/motd/tips/10-tips.md

```diff
@@ -11,3 +11,4 @@ Use `brew search` and `brew install` to install packages. Bluefin will take care
 Use `Ctrl`-`Alt`-`Enter` to quickly open a terminal
 Use `Super`-`Space` to run Search Light (Super is your Windows key!)
 Tailscale is included, check out [their docs](https://tailscale.com/kb/1017/install)
+Bluefin is your gateway to AI and LLMs 🦙 `ujust ollama` to [get started](https://ollama.com/)
```

Comment:
No idea if this emoji will work.  It actually shows in GitHub for me, unlike the others
<img width="263" alt="image" src="https://github.com/ublue-os/bluefin/assets/46304672/0a808ef1-38dc-4286-81d2-bf5b4545a552">
<img width="263" alt="image" src="https://github.com/ublue-os/bluefin/assets/46304672/2ad32ea7-101c-4c00-a302-61b216d750d5">

Reply:
I'll just remove it for now.